en/docs/install-and-setup/configuring-api-manager-for-ob.md

```diff
@@ -263,7 +263,20 @@ account retrieval. By default, this is disabled and the configuration is set to
     password="$ref{super_admin.password}"	
     server_url = "{tcp://<SI_HOST>:7612}"	
     ```  
-   
+
+16. If you are using WSO2 API Manager 4.2.0, you need to change the API Manager REST API version from V2 to V3.
```

Comment:
AFAIK in the default configs the version is not mentioned. So we will not be able to say to change it. But we can mention to use a the relevant endpoint version based on the APIM version. @Ashi1993 WDYT?

Reply:
Default value is read from the J2 file hence we dont have to menion the default values.